.github/workflows/performance.yml

```diff
@@ -32,17 +32,19 @@ permissions: {}
 jobs:
   # Runs the performance test suite.
   performance:
-    name: Performance tests ${{ matrix.memcached && '(with memcached)' || '' }}
-    uses: WordPress/wordpress-develop/.github/workflows/reusable-performance.yml@trunk
+    name: Performance tests${{ matrix.multisite && ' (Multisite)' || '' }}${{ matrix.memcached && ' (with memcached)' || '' }}
+    uses: swissspidy/wordpress-develop/.github/workflows/reusable-performance.yml@try/performance-tests-expand
```

Comment:
I assume this will be reverted prior to commit?

Reply:
Yes of course. Only required for testing.